setup/storage/minio.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: single
+title:  "Minio"
+sidebar:
+  nav: setup
+---
+
+[Minio](https://www.minio.io/) is an S3-compatible object store that you can
+host yourself. This is the persistent storage solution we recommend when you
+don't want to depend on a cloud provider to host your Spinnaker data. However,
+losing Minio's data will mean losing all your Spinnaker application metadata,
```

Comment:
Maybe break out this last sentence, "However, losing ....pipelines" into a warning paragraph.

Reply:
Good point